src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Json/GraphQLNamedQueryRequestJsonConverter.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using GraphQL;
+
+namespace OrchardCore.Apis.GraphQL.Json;
+public class GraphQLNamedQueryRequestJsonConverter : JsonConverter<GraphQLNamedQueryRequest>
+{
+    public static GraphQLNamedQueryRequestJsonConverter Instance = new();
+    /// <summary>
+    /// Name for the operation name parameter.
+    /// See https://github.com/graphql/graphql-over-http/blob/master/spec/GraphQLOverHTTP.md#request-parameters
+    /// </summary>
+    private const string OPERATION_NAME_KEY = "operationName";
```

Comment:
Why the uppercase and the underscore? It should be `OperationNameKey` and the same goes for all of the 
```suggestion
    private const string OperationNameKey = "operationName";
```

Reply:
> Why the uppercase and the underscore? It should be `OperationNameKey` and the same goes for all of the

These are copied from the `Graphql.SystemJson` package